components/private/ThumbTrack/src/MDCNumericValueLabel.h

```diff
@@ -14,18 +14,25 @@
 
 #import <UIKit/UIKit.h>
 
-@interface MDCNumericValueLabel : UIView
+@interface MDCNumericValueLabel : UIView <UIContentSizeCategoryAdjusting>
 
 /** The background color of the value label. */
-@property(nonatomic, retain) UIColor *backgroundColor;
+@property(nonatomic, retain, nonnull) UIColor *backgroundColor;
```

Comment:
Tip: Nit: Since you're updating the property semantics, could you switch `retain` with `strong`? I think `retain` is now considered archaic.

Reply:
done